qa/testing.go

```diff
@@ -190,34 +196,84 @@ func (f ResourceFixture) setDatabricksEnvironmentForTest(client *common.Databric
 	}
 }
 
-// Apply runs tests from fixture
-func (f ResourceFixture) Apply(t *testing.T) (*schema.ResourceData, error) {
+func (f ResourceFixture) validateMocks() error {
+	isMockConfigured := f.MockAccountClientFunc != nil || f.MockWorkspaceClientFunc != nil
+	isFixtureConfigured := f.Fixtures != nil
+	if isFixtureConfigured && isMockConfigured {
```

Comment:
In some cases, we may not use API for everything - I don't know if it will be possible to have both mocks together?